pkg/kudoctl/kudoinit/setup/setup.go

```diff
@@ -21,6 +21,9 @@ var _ kudoinit.InstallVerifier = &Installer{}
 type Installer struct {
 	options kudoinit.Options
 	steps   []kudoinit.Step
+
+	managerStep *manager.Initializer
```

Comment:
really don't like the name... step is a meaningful thing in kudo, and this doesn't align... it would be ok if it was some kind of step, but I don't see that... it is a manager Initializer..

Reply:
I've renamed it to `managerInitializer`, same for the webhook